ivy/functional/frontends/torch/nn/functional/loss_functions.py

```diff
@@ -191,3 +192,23 @@ def nll_loss(
     ret = reduct(loss)
 
     return ret
+
+
+@to_ivy_arrays_and_back
+def cosine_embedding_loss(
+    input1,
+    input2,
+    target,
+    margin=0.0,
+    size_average=None,
+    reduce=None,
+    reduction="mean",
+):
+    similarity = distance_functions.cosine_similarity(input1, input2)
+    if target == 1:
+        loss = 1 - similarity 
```

Comment:
This line throws an error as `TypeError: unsupported operand type(s) for -: 'int' and 'Tensor'`. Perhaps using a debugger to see why the operation isn't supported. Normally an operation as such should be valid. Feel free to ask anything and please request when my review when fixed.